models/product.model.js

```diff
@@ -1 +1,34 @@
-//created file
\ No newline at end of file
+//created file
+const mongoose = require('mongoose');
+
+const ProductSchema = new mongoose.Schema(
+	{
+		name: {
+			type: String,
+			required: true,
+		},
+		sku: {
+			type: String,
+			required: true,
+			unique: true,
+		},
+		quantity: {
+			type: Number,
+			default: 0,
+		},
+		createdBy: {
+			type: String,
+			required: true,
+		},
+		createdAt: {
+			type: Date,
+			required: true,
+		},
+		path: {
```

Comment:
what does path signify ? I thought we use the ID of the uploaded file for images. @iamNanaKwame